### ISSUE TITLE
copy_table raises argument "body" is missing, with no default error

### ISSUE THREAD

**Issue by hidekoji** [r-dbi/bigrquery#172]
R: 3.4.1
bigrquery:  0.4.1

``` r
library(bigrquery)

project_id <- "exploratory-io"
sourceDatasetId <- "exploratory_dataset"
sourceTableId <- "exploratory_bq_preview_table"
projectId <- "exploratory-io"
targetProjectId <- "exploratory-io"
targetDatasetId <- "dataset_for_backup"
targetTableId <- "test_table"
src <- list(project_id = projectId, dataset_id = sourceDatasetId, table_id = sourceTableId)
dest <- list(project_id = targetProjectId, dataset_id = targetDatasetId, table_id = targetTableId)
bigrquery::copy_table(src, dest)
#> Error in stopifnot(is.list(x), is.list(val)): argument "body" is missing, with no default
```

**Comment by meridithperatikos** [comment]
I am having the same issue, thanks!

**Comment by RushabhSavla** [comment]
Same issue here as well. @hidekoji : Did you find any resolution to this issue?

**Comment by byapparov** [comment]
This seems to be same #200, it is fixed here #208
